NETDefault/Gateway.Buckaroo/DependencyConfiguration.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Microsoft.Extensions.DependencyInjection;
+
+namespace Gateway.Buckaroo;
+
+public class DependencyConfiguration
+{
+    public static void Register(IServiceCollection services)

```

Comment:
Perhaps we should use Bas' suggestion and try to make these Register methods extension methods, and use it like `sevices.AddBuckaroo()`.

Reply:
I would also opt to move the `AddOptions<>` of `BuckarooConfiguration` to this method as well, and require `IConfiguration` here. This simplifies everything, and its not like anyone is going to use it any other way.